wsmaster/che-core-api-workspace-shared/src/main/java/org/eclipse/che/api/workspace/shared/Constants.java

```diff
@@ -68,6 +68,13 @@
   public static final String SERVER_EXEC_AGENT_WEBSOCKET_REFERENCE = "exec-agent/ws";
 
   public static final String WS_AGENT_PORT = "4401/tcp";
+  public static final String WS_AGENT_TRACK_CONNECTION_SUBSCRIBE =
```

Comment:
This class is not related to wsagent. Consider moving these constants elsewhere.

Reply:
Constants are moved in shared module to have ability use them on client side and on server side